.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,31 +1,55 @@
 ---
 name: "\U0001F41B Bug report"
 about: Create a report to help us fix things
+title: ''
+labels: ['bug', 'needs-triage']
+assignees: ''
+projects: ['nightscout/2']
 
 ---
+## Describe the bug
+*A clear and concise description of what the bug is. Describe what you see versus what you expect to see.*
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+## Attach a Log
+*Tap the Trio settings icon at the bottom of the screen, then tap 'Share logs' on the bottom of the list and attach it to this ticket.*
 
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+## To Reproduce
+*Steps to reproduce the behavior:*
+1. *Go to '...'*
+2. *Click on '....'*
+3. *Scroll down to '....'*
+4. *See error*
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+## Expected behavior
+*A clear and concise description of what you expected to happen.*
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+## Screenshots
+*If applicable, add screenshots to help explain your problem.*
 
-**Smartphone (please complete the following information):**
+## Setup Information (please complete the following information):
 
-**Setup Information (please complete the following information):**
-* Pump type
-* CGM type and CGM app
-* Trio version, branch and git reference (see Trio Settings)
+### Smartphone:
+* Hardware: *[e.g. iPhone 15 Pro]*
+* OS Version: *[e.g. iOS 17.5]*
 
-**Additional context**
-Add any other context about the problem here.
+### Pump:
+* Manufacturer: *[e.g. Insulet]*
+* Model: *[e.g. Omnipod Dash or Eros]*
+
+### CGM:
+* Device: *[e.g. Dexcom G7]*
+* Manager app: *[e.g. Dexcom App or xDrip4iOS]*
+
+### Trio Version:
+* Version Number: *[e.g. 1.9.2]*
+* Repo: *nightscout/trio*
+* Git Reference: *[e.g. commit hash]*
+
+## Technical Details
+*If applicable, provide any technical details that might help in diagnosing the problem. This could include logs, error messages, or relevant configuration details.*
+
+## Additional context
+*Add any other context about the problem here.*
+
+## Severity Level
+*(Optional) Indicate the severity level of the bug (e.g., minor, major, critical) based on its impact on the Trio app functionality.*
```

Comment:
A normal user can’t and actually should not be able to assign a severity index or level. Most users will make every single issue major or critical (speaking from experience at work 😂). I’d strike this section. Project managers in our end that triage tickets will do that 😊

Reply:
Agreed